CONTRIBUTING.md

```diff
@@ -59,9 +55,28 @@ If your code requires new dependencies, be sure that those are available on all
 
 ### User experience
 
-Jamulus is used by people all over the world with different backgrounds and knowledge. Features and language should be user focused.
-The Jamulus project has a [style and tone guide](https://jamulus.io/contribute/Style-and-Tone) which you should follow to remain consistent.
-Features should be usable in the sense that they act as expected to somebody who does not have a technical background.
+Jamulus is used by people all over the world with different backgrounds and knowledge. Therefore, think user focused.
+
+- Features should be usable in the sense that they act as expected to somebody who does not have a technical background.
+- Make sure you follow the [style and tone guide](https://jamulus.io/contribute/Style-and-Tone) to remain consistent.
+
+### Submitting code
+
+- Create a Github fork of the Jamulus code and create a Pull Request when you are done. Please don't use diff/patch to send your code changes. GitHub can be confusing, so feel free to ask for help.
+- Please run a local (build/feature) test to ensure there are no errors. After opening a pull request, keep an eye on the CI checks for quality or compile issues and fix them as required.
+
+#### Updating a PR/fixing conflicts
+
+If there are conflicts with jamulussoftware/jamulus showing when you raise your pull request or are requested to "rebase" your work, do so as follows:
+
+```shell
+git checkout master ;# checkout your local master
+git pull upstream master --tags ;# refresh from jamulussoftware/jamulus (i.e. use your upstream remote name. If you haven't set it up yet, add it via 'git remote add upstream git@github.com:jamulussoftware/jamulus')
```

Comment:
This only works when a remote with the name `upstream` has been created. This isn't explained anywhere in this document, as far as I can see?
(I'm well aware that this part is not new in this PR and has just been moved around...)

Reply:
This PR is a general one, so improvement is good. 
Adding another line should be enough.

Reply:
Ok. Please see the last change

Reply:
Sounds good, thanks.